camsrv/f5wfs.py

```diff
@@ -0,0 +1,166 @@
+"""
+MMTO F/5 WFS camera interface
+"""
+
+import os
+import socket
+import time
+import pkg_resources
+
+import tornado
+import tornado.web
+import tornado.httpserver
+import tornado.ioloop
+import tornado.websocket
+from tornado.log import enable_pretty_logging
+
+from pathlib import Path
+
+import logging
+import logging.handlers
+
+from astropy.io import fits
+import io
+
+from indiclient.indicam import SimCam, F9WFSCam
+
+from .sbig import SBIGClient
+from pyindi.webclient import INDIWebApp
+import base64
+
+from .camsrv import CAMsrv
+
+enable_pretty_logging()
+logger = logging.getLogger("")
+logger.setLevel(logging.INFO)
+log = logging.getLogger('tornado.application')
+import asyncio
+log.setLevel(logging.INFO)
+
+
+from .header import update_header
+
+F9WFSPORT = 8787
+
+__all__ = ['F5WFSsrv', 'main']
+
+
+
+
+class F5WFSsrv(CAMsrv):
+    class WFSModeHandler(tornado.web.RequestHandler):
+        """
+        Configure CCD to be in WFS mode, square with 3x3 binnind
+        """
+        def get(self):
+            cam = self.application.camera
+            log.info("Configuring f/9 WFS camera for WFS observations...")
+            cam.wfs_config()
+
+    class DefaultModeHandler(tornado.web.RequestHandler):
+        """
+        Configure CCD to be in WFS mode, square with 3x3 binnind
+        """
+        def get(self):
+            cam = self.application.camera
+            log.info("Setting f/9 WFS camera to its default configuration, full-frame with 1x1 binning...")
+            cam.default_config()
+
+    def connect_camera_old(self):
```

Comment:
This method is not necessary. The camera connection is handled by the javascript in f5wfs.html. There is also a simulation button there.